portal/src/components/Materials/SortableMaterials/SortableMaterials.component.html

```diff
@@ -0,0 +1,141 @@
+<section class="collection__materials">
+  <slot name="header-info"></slot>
+  <draggable
+    v-model="myList"
+    draggable=".materials__item"
+    class="materials__items deleting"
+    :class="{
+      loading: current_loading,
+      list: itemsInLine === 1,
+      tile: itemsInLine > 1
+    }"
+  >
+    <div
+      v-for="material in myList"
+      :class="[
+        `tile--items-in-line-${itemsInLine}`,
+        `materials__item--items-in-line-${itemsInLine}`,
+        'select-delete'
+      ]"
+      :key="material.external_id"
+      class="materials__item tile"
+    >
+      <div class="materials__bookmark" v-if="material.has_bookmark">Bookmark</div>
+      <button
+        class="select-icon"
+        v-if="contentEditable"
+        @click="deleteFromCollection(material)"
+      >
+        Delete
```

Comment:
Moet dit geen vertaling zijn? Of zie je dit toch niet staan, maar het icoon? (kan het dan niet weg?)

Reply:
je ziet dit inderdaad niet staan. ik zal het weghalen